### ISSUE TITLE
Sync a remote directory (network folder shared)

### ISSUE THREAD

**Issue by camlafit** [haiwen/seafile#1163]
Hi

With Windows have try to sync a remote folder provided by a nas. Seafile client accept to target remote directory (//nas/directory/) but do anything.
If this remote directory is mapped as a network drive, seafile accept to index and syncrhonize data)

On smal society, looks regular to provide a nas or network drive with many directories dedicated to each empployed/computer.

Is it possible to allow to target directly a remote shared directory ?

**Comment by shoeper** [comment]
There have been dozens of issues describing this. But it is no Seafile problem. It is just not possible to monitor remote directories properly.
Only option you have is to make a rightclick and click on sync now.

**Comment by camlafit** [comment]
hi

Before to open this issues, I searched in olders with differents keywords
and didn't see anything. Sorry about this duplicate.

Is not possible in this case to have a cron to do a check on every 5' (or
other period). And before do a warning about this "degraded" mode if a
network target is detected .

Thanks

**Comment by shoeper** [comment]
This would be possible, but it may cause new issues
* high load
* permanent scanning when many data is on network target
* high network drive load when many people are doing this
* -> leads to heavy battery drain on latops
* reports that data is not synced instantly

Maybe there are even more issues. I also don't really get what this is useful for.

**Comment by camlafit** [comment]
Hi

I understand the problem, in this case maybe add a warning about a network
directory (with precision you mus sync manually) or forbid network sync.
In this case the pb looks from seafile because users don't know about
theses technical limitation.

**Comment by petergeier** [comment]
Well - I had the same challenge here but I found 2 workarounds:

1. The easiest way is to just start one Seafile client on the main fileserver (then the OS will be notified about any file changes) - this is the way I went for now until this is fixed on the server. The main company libraries are shared FROM HERE and as the directories will be on local disks.

2. More involved and a pain in the neck to setup: Use a Seafile client on a special PC on the local network (e.g. VM) which is always on. Sync the libraries to a local HD. Now use a local filesync tool (such as goodsync or some of the free ones e.g. https://github.com/haiwen/seafile/issues/863 ) and auto-sync the network files to the local library every xx time unit.

Both work, tested them.
